app/views/v2/pkg/show.mustache

```diff
@@ -0,0 +1,147 @@
+{{! Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
We should consider adding golden files for the v2 templates as well.

Reply:
added